pkg/cmd/server/admin/create_bootstrappolicy_file.go

```diff
@@ -32,15 +33,15 @@ type CreateBootstrapPolicyFileOptions struct {
 	OpenShiftSharedResourcesNamespace string
 }
 
-func NewCommandCreateBootstrapPolicyFile() *cobra.Command {
+func NewCommandCreateBootstrapPolicyFile(commandName string, baseName string, out io.Writer) *cobra.Command {
```

Comment:
> On Mar 26, 2015, at 10:40 AM, David Eads <notifications@github.com> wrote:
> 
> In pkg/cmd/server/admin/create_bootstrappolicy_file.go:
> 
> > @@ -32,15 +33,15 @@ type CreateBootstrapPolicyFileOptions struct {
> >  	OpenShiftSharedResourcesNamespace string
> >  }
> >  
> > -func NewCommandCreateBootstrapPolicyFile() *cobra.Command {
> > +func NewCommandCreateBootstrapPolicyFile(commandName string, baseName string, out io.Writer) *cobra.Command {
> And if you were passing information to an API, you wouldn't want to pass FullName, you'd pass FirstName LastName (or GivenName SurName).
> 
> Should args be: "create-master-certs", "openshift admin create-master-certs" or "create-master-certs", "openshift admin". I vote for the latter and some commands are taking that, but calling the second arg "fullName" in spite of it being a "baseName".
> 
If you do that, every command has to do string manipulation.  Stick to fullName name, but if you have a better naming scheme, do so.

> —
> Reply to this email directly or view it on GitHub.
>